src/eureka/S5_lightcurve_fitting/plots_s5.py

```diff
@@ -188,8 +188,8 @@ def plot_corner(samples, lc, meta, freenames, fitter):
     - December 29, 2021 Taylor Bell
         Moved plotting code to a separate function.
     """
-    samples = np.copy(samples)
-    freenames = np.copy(freenames)
+    samples = samples
```

Comment:
What is the point of these two lines (191+192)?

Reply:
Hah, indeed - I'd meant to remove the np.copy which was unnecessary and RAM intensive, but had not meant to leave those lines behind